backend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG PYTHON_VERSION=3.11
```

Comment:
This just complexifies stuff without added value

Reply:
I was thinking that since the minimum python version is 3.11, an admin can build an image using 3.12 through build args. I will update it in the next commit.

Reply:
I understand but deployment is already an effort in itself and we don't want to add constraints and possibilities there.